src/imapcommands.c

```diff
@@ -2405,7 +2405,20 @@ static void _ic_store_enter(dm_thread_data *D)
 	if (self->ids_list) {
 		GString *failed_ids = g_list_join_u64(self->ids_list, ",");
 		buffer = p_string_new(self->pool, "");
-		p_string_printf(buffer, "MODIFIED [%s]", failed_ids->str);
+		Field_T val;
+		GETCONFIGVALUE("compatibility_report_skip_failed", "IMAP", val);
```

Comment:
Since this is a new option, it should be included in the default dbmail.conf

Also, I'm not happy with  the name. disable_idle_report or something similar would better describe the purpose, imo.